include/envoy/http/filter.h

```diff
@@ -268,6 +268,22 @@ class StreamFilter : public StreamDecoderFilter, public StreamEncoderFilter {};
 typedef std::shared_ptr<StreamFilter> StreamFilterPtr;
 
 /**
+ * Stream access log handler interface.
+ */
+class AccessLogHandler {
+public:
+  virtual ~AccessLogHandler() {}
+
+  /**
+   * Called with request_info data.
+   * @param request_info: the request_info data.
+   */
+  virtual void logRequestInfo(const Http::AccessLog::RequestInfo& request_info) PURE;
```

Comment:
can we also pass request/response headers (optionally) like the other access log calls? Might be useful for others in the future.